react-native-image-editor.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "react-native-netinfo"
```

Comment:
This should be react-native-image-editor

Reply:
Done